torch/nn/parallel/distributed.py

```diff
@@ -1835,6 +1838,12 @@ def _check_comm_hook(self, hook):
     def _distributed_rank(self):
         return dist.get_rank(self.process_group)
 
+    @staticmethod
+    def _get_data_parallel_params(module):
```

Comment:
is it DP or DDP that's being talked about?

if DDP, then having only `data_parallel` in the function name might be confusing